docker/test-server/Dockerfile

```diff
@@ -6,12 +6,12 @@ ENV PIP_DISABLE_PIP_VERSION_CHECK=1
 ENV PIP_NO_CACHE_DIR=1
 
 # install packages and pip requirements first, in a single step,
-COPY /requirements.txt /requirements.txt
+COPY /requirements.in/base.txt /requirements.txt
 RUN set -ex ;\
     apk add libpq postgresql-dev --virtual .build-deps coreutils gcc libc-dev libffi-dev make ;\
     pip install -U pip ;\
     pip install --no-cache-dir --progress-bar off -r /requirements.txt ;\
-    apk del .build-deps make gcc coreutils ;\
+    # apk del .build-deps make gcc coreutils ;\  # do not uncomment this, otherwise errors start producing.
```

Comment:
Why should we comment this line? If it is from the original repo, it is probably an important step.

Reply:
deleting these build-deps files causing docker to failed, it was showing error of missing libraries (and those libs were included in the docker file), so there was no point to run docker without commenting this line.

Reply:
You mean media files failure to AWS S3? I don't think so, it has to do any thing with that.